pkg/config/config.go

```diff
@@ -135,6 +135,7 @@ type MetricConfig struct {
 	ValueType          string                    `yaml:"type"`
 	ConstantLabels     map[string]string         `yaml:"const_labels"`
 	StringValueMapping *StringValueMappingConfig `yaml:"string_value_mapping"`
+	MQTTValueScale     uint32                    `yaml:"mqtt_value_scale"`
```

Comment:
Why only an unsigned int? what, if I want to invert the value? Why only an integer? How about usecases where I want to scale it lower. e.g. unit used is kilograms, but you want to use only grams in your system.

Reply:
Makes sense, I was biased by a common use case we have in Sysdig; let's make it a `float64`

Reply:
Thank you!